binaries/pip/build_wheels.sh

```diff
@@ -57,7 +53,13 @@ mkdir output
 
 install_java_deps
 
-install_torch_deps
+cd ../../
+
+set +u
+install_torch_deps $1
```

Comment:
Please update the readme with details of the parameter to pass for $1

Reply:
`torch` packages are not required for building the `torchserve` binaries. Refactored the code.